website/static/getting-started.html

```diff
@@ -102,7 +102,7 @@ <h3>A simple todo store...</h3>
 	report() {
 		if (this.todos.length === 0)
 			return "<none>";
-		return `Next todo: "${this.todos[0].task}". ` +
+		return `Next todo: "${this.todos.find(todo => todo.completed === false).task}". ` +
```

Comment:
`find` will return `undefined` if nothing is found. In this case, it doesn't matter that much, but let's show a proper code here that won't break in less contrived situations.

Reply:
What approach should I use?

My preference:
```
const nextTodo = this.todos.find(todo => todo.completed === false);
return `Next todo: "${nextTodo ? nextTodo.task : "n/a"}". `
```

Is that fine? Or does the code style for this project call for something different?

Some other options:
```
2) return `Next todo: "${(this.todos.find(todo => todo.completed === false) || {}).task || "n/a"}". `
```
```
3) return `Next todo: "${this.todos.filter(todo => todo.completed === false).map(todo => todo.task)[0] || "n/a"}". `
```

Reply:
Yea, I would go with the first one. There is no project preference I am aware off :)

Reply:
Updated the PR with the fix. (the first option -- though with "<none>" as text instead of "n/a" to match the string above it)